dashboard/docs/react-developer-guide.md

```diff
@@ -99,8 +99,8 @@ You can find the most recent version of this guide [here](https://github.com/fac
 
 Create React App is divided into two packages:
```

Comment:
It was modified by prettier. However, since new react apps created with CRA are not including this file, should we remove it?

Reply:
yep, this is probably outdated, we can remove it.